tests/nix_tests/configs/osmosis-outpost.jsonnet

```diff
@@ -0,0 +1,254 @@
+local config = import 'default.jsonnet';
+
+config {
+  'evmos_9000-1'+: {
+    'account-prefix': 'evmos',
+    'coin-type': 60,
```

Comment:
Why don't we have this already configured in the underlying base `evmos_9000-1` configuration?

Reply:
it is not included in the default config ATM, but will refactor this to include it